packages/main/src/TimePicker.hbs

```diff
@@ -0,0 +1,73 @@
+<div id="{{this._id}}" formatPattern="{{formatPattern}}" class="ui5-timepicker">
+	<ui5-input
+			id="{{_id}}-inner"
+			value="{{value}}"
+			?disabled="{{disabled}}"
+			?readonly="{{readonly}}"
+			value-state="{{valueState}}"
+			@ui5-change="{{_handleInputChange}}"
+			@ui5-input="{{_handleInputLiveChange}}"
+	>
+		<ui5-icon
```

Comment:
the icon does not get the expected hover effect: different background and cursor pointer

Reply:
Done